test/chpldoc/compflags/combinations/version.doc.prediff

```diff
@@ -2,3 +2,4 @@
 CWD=$(cd $(dirname $0) ; pwd)
 $CWD/../../../compflags/bradc/printstuff/versionhelp.sh $@ > ${1}.good
 $CWD/../../../compflags/bradc/printstuff/copyrighthelp.sh $@ >> ${1}.good
+sed -e "/built with LLVM version/d" -i ${1}.good
```

Comment:
Why is this change in this PR? I thought we were going to stop printing this part of the `--version`

Reply:
Oh are we sharing a .good file between chpl and chpldoc? Probably should stop doing that.

Reply:
Could we make versionhelp-chpldoc.sh e.g. and use that in these different cases instead? and it could leave out the LLVM line?

Reply:
yes, I think it's just the version ones that need a different output anyway